services/svc_googlephotos.py

```diff
@@ -201,6 +201,9 @@ def getUrlFromImages(self, types, displaySize, images):
       proposed = images[index]['baseUrl']
       if self.memorySeen(proposed):
         continue
+      if not self.isCorrectOrientation(images[index]['mediaMetadata'], displaySize):
```

Comment:
This should be after `self.memoryRemember(proposed)` since it will allow other parts of the code to be aware of the skipped image (granted, nothing does this today, but at least it's prepared)

Reply:
done